src/main/groovy/com/bugsnag/android/gradle/BugsnagVariantOutputTask.groovy

```diff
@@ -135,6 +135,11 @@ class BugsnagVariantOutputTask extends DefaultTask {
             apiKey = project.bugsnag.apiKey
         } else {
             apiKey = getManifestMetaData(metaDataTags, ns, BugsnagPlugin.API_KEY_TAG)
+
+            if (apiKey == null) {
+                project.logger.warn("Could not find '$BugsnagPlugin.API_KEY_TAG' " +
```

Comment:
There's a mix of levels that warnings are being logged at - in `readManifestFile` or in one of the `get` methods. In fact, I believe it would log twice if API weren't found?

I suggest all these logger calls be made from `readManifestFile`.

Reply:
Thanks - this is a good point. I've consolidated all the warnings to be logged in `readManifestFile` instead.